sdk/python/kfp/dsl/_pipeline.py

```diff
@@ -60,6 +60,8 @@ def __init__(self):
     self.image_pull_secrets = []
     self.timeout = 0
     self.ttl_seconds_after_finished = -1
+    self.min_available = None
+    self.selector = None
```

Comment:
We might want to make these fields private (leading underscore).
Also I think we could use better names. `Selector` is too generic.
Maybe `_pod_disruption_budget_selector` and `_pod_disruption_budget_min_available`.

Reply:
Yeah, make sense will update.